apps/academy/src/pages/tracks/nft-solidity/5.mdx

```diff
@@ -129,35 +129,41 @@ Ok to proceed? (y) y
 ✔ Would you like to use `src/` directory? … Yes
 ✔ Would you like to use App Router? (recommended) … Yes
 ✔ Would you like to customize the default import alias (@/*)? … Yes
+✔ What import alias would you like configured? … @/*
 Creating a new Next.js app in ((path)):/developerdao/frontend.
 ```
 
-Once the setup is complete, we open the `frontend` directory and get cracking:
+Once the setup is complete, we open the `frontend` directory and get cracking.
+Try running `npm run dev` to see if everything is working as expected.
 
 ```bash
 cd frontend
+npm run dev
 ```
 
+You'll see a message that says which url to view your site at.
+It should say <strong>http://localhost:3000</strong>
+
 <br />
 
 ---
 
-## The Front-end
+## The Frontend
```

Comment:
Hehe, we had a conversation about this. The final decision was to use front-end hyphenated in texts, headers and descriptions, and `frontend` in building the dApp in the lesson, or building Academy's architecture.

Reply:
I switched this back to `Front-end`